### PR TITLE
Add provisioned instances to catalog view

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Adds info about service instances in new tab in catalog view
- Adds two new components: instance status & tab elements counter

**Related issue(s)**
See #587

### CODE REVIEW

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js [hunk 1]
```diff
@@ -319,6 +354,19 @@ class ServiceClassTabs extends Component {
...
+              aditionalStatus={this.getTabElementsIndicator(
+                this.props.serviceClass.instances.length,
+              )}
+              title={'Instances'}
+            >
+              <Table
+                headers={['INSTANCE', 'STATUS']}
+                tableData={this.prepareRowData(
+                  this.props.serviceClass.instances,
+                )}
```
Comment (akucharska) [inline catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: add 'No Service Instances found' in src/variables.js and use it here:

import { serviceClassConstants } from '../../variables';
notFoundMessage={serviceClassConstants.emptyInstancesListMessage}

**Comment by akucharska** [comment]
One more thing. When ServiceClassTabs recieve error (from documentation fetching) whole Tabs section disappear.

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js [hunk 2]
```diff
@@ -319,6 +354,19 @@ class ServiceClassTabs extends Component {
                 <ODataReact schema={odata.source || deprecatedOdataSpec} />
               </Tab>
             ) : null}
+            <Tab
+              aditionalStatus={this.getTabElementsIndicator(
+                this.props.serviceClass.instances.length,
+              )}
+              title={'Instances'}
```
[the remark this answers is not in the document]
Comment (y-kkamil) [reply catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: fixed

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js [hunk 3]
```diff
@@ -319,6 +354,19 @@ class ServiceClassTabs extends Component {
...
+            >
+              <Table
+                headers={['INSTANCE', 'STATUS']}
```
[the remark this answers is not in the document]
Comment (y-kkamil) [reply catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: i vote for instance name ;) corrected.

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js (hunk 1 above)
Comment (y-kkamil) [reply catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: done

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js [hunk 4]
```diff
@@ -319,6 +354,19 @@ class ServiceClassTabs extends Component {
                 <ODataReact schema={odata.source || deprecatedOdataSpec} />
               </Tab>
             ) : null}
+            <Tab
+              aditionalStatus={this.getTabElementsIndicator(
+                this.props.serviceClass.instances.length,
+              )}
+              title={'Instances'}
+            >
+              <Table
```
[the remark this answers is not in the document]
Comment (y-kkamil) [reply catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: great idea, extracted.

**Pull request merged**